utils/zerovec/src/lib.rs

```diff
@@ -371,16 +374,19 @@ pub use zerovec_derive::make_ule;
 /// to convert the [`VarULE`] type back to this type in a cheap, zero-copy way (see the example below
 /// for more details).
 ///
-/// Provided the type implements [`serde::Serialize`](dep_serde::Serialize) and [`serde::Deserialize`](dep_serde::Deserialize), this attribute can also generate
-/// the relevant serialize/deserialize implementations for the [`VarULE`] type if you apply the `#[zerovec::serde]`
-/// attribute. Those impls are required to support human-readable serialization of the VarZeroVec.
+/// Certain custom derives can be inherited to the [`VarULE`] type with `#[zerovec::derive(...)]`. For example, if the type has a `Debug` impl,
```

Comment:
Suggestion: Make a clearer list of the things that are derived and need to be opted out, and the things that are available and can be opted in. Something like,

```
#[make_varule] will automatically apply the following derives to the `VarULE` type:

- [`Ord`]
- [`PartialOrd`]
- [`ZeroMapKV`]
- (anything else?)

To disable one of the automatic derives, use: `#[zerovec::skip_derive(ZeroMapKV)]`

The following derives are available, but not automatic:

- [`Debug`]
- [`Serialize`]
- [`Deserialize`]
- (anything else?)

To enable one of these additional derives, use: `#[zerovec::derive(Debug)]`
```

Also apply a similar change to `#[make_ule]`.